config-templates/config.php

```diff
@@ -55,6 +55,11 @@
     'technicalcontact_email' => 'na@example.org',
 
     /*
+     * Sender email should use the same domain as your application domain to prevent SPF record filtering
```

Comment:
Probably this should read something like "The envelope from address for outgoing emails. This should be in a domain that has your application's IP addresses in its SPF record to prevent it from being rejected by mail filters."